bin/all_sky_search/pycbc_fit_sngls_over_multiparam

```diff
@@ -15,8 +15,64 @@
 from __future__ import division
 
 import sys, h5py, argparse, logging, pycbc.version, numpy
+from scipy.stats import norm
 from pycbc.events import triggers
 
+def dist(i1, i2, parvals, smoothing_width):
+    dsq = 0
+    for v, s in zip(parvals, smoothing_width):
+        dsq += (v[i2] - v[i1]) ** 2.0 / s ** 2.0
+    return dsq ** 0.5
+
+def smooth_templates(nabove, invalphan, ntotal, template_idx,
+                      weights=None):
+    if weights is None: weights = numpy.ones_like(template_idx)
+    nabove_t_smoothed = numpy.average(nabove[template_idx], weights=weights)
+    ntotal_t_smoothed = numpy.average(ntotal[template_idx], weights=weights)
+    invalphan_mean = numpy.average(invalphan[template_idx], weights=weights)
```

Comment:
maybe add a comment here to explain that the max likelihood fit for 1/alpha is linear in the trigger statistic values, so we perform an average of (n_above / alpha) over templates and then invert this and multiply by (smooth) nabove to obtain smoothed alpha

Reply:
Added in docstring explanation of the function